x/valset/keeper/keeper.go

```diff
@@ -673,26 +714,19 @@ func (k Keeper) externalChainInfoStore(ctx sdk.Context, val sdk.ValAddress) sdk.
 	)
 }
 
-func (k Keeper) _externalChainInfoStore(ctx sdk.Context) sdk.KVStore {
-	return prefix.NewStore(
-		ctx.KVStore(k.storeKey),
-		[]byte("external-chain-info"),
-	)
+func (k Keeper) _externalChainInfoStore(ctx context.Context) storetypes.KVStore {
+	store := runtime.KVStoreAdapter(k.storeKey.OpenKVStore(ctx))
+	return prefix.NewStore(store, []byte("external-chain-info"))
 }
 
-func (k Keeper) snapshotStore(ctx sdk.Context) sdk.KVStore {
-	k.Logger(ctx).Debug("snapshot store", "store-key-name", k.storeKey.Name(), "store-key-string", k.storeKey.String())
-	return prefix.NewStore(ctx.KVStore(k.storeKey), []byte("snapshot"))
+func (k Keeper) snapshotStore(ctx context.Context) storetypes.KVStore {
+	store := runtime.KVStoreAdapter(k.storeKey.OpenKVStore(ctx))
+	k.Logger(ctx).Debug("snapshot store", "store-key-name", "store-key-string", k.storeKey)
```

Comment:
Why would you leave the field key and only remove value? Now the whole ordering doesn't work anymore.

Just remove the entire statement, I don't think it's needed.

Reply:
ok